### ISSUE TITLE
Detect non-activated accounts

### ISSUE THREAD

**Issue by JustMyGithub** [kiideveloper/pokiimap#88]
I noticed today that this great tool does not detect whether an account is actually activated.
From what I know in order to use an account for scnanning you need to accept the terms of use. I'm not sure whether you have to do the tutorial, catch your first pokiimon and choose a name as well.

however, when I added an account which has not accepted the ToS (in-Game), Pokiimap 1.6.0 did not detect that this is the case.

**Comment by hardknoxni69a** [comment]
almost all bots are able to be used without acceting tos i have a few accounts that didnt set up TOS and at level 35 i was able to login on device and play fine with no bans at all and this account was botted for over month without accepting TOS not to mention it is a scan accounts so who cares if it has accepted TOS

**Comment by JustMyGithub** [comment]
You do care, if this causes the scan to return empty results. I know that this was the case using an other scanning app.
And really, screw you for using bots. People like you are destroying the game....

**Comment by hardknoxni69a** [comment]
i would agree with you if it was true tos will not change scan results as i have not logged into any of my scan accounts. maybe other apps are different bot not this one. and yes some people who bot do ruin the game but if little girls that cant play complain thats not my fault i played my ass off till level 31 and some ass reported me for cheating when i never did once and i wasnt going to play 3 months of work again so i started to bot but im a fair guy now that im at 34 i just try to get the best of everything and got my pokedex filled but i dont play in gyms because people are sore losers if little girls werent jealous of a dedciated player at 31 i wouldnt have botted in first place
